### ISSUE TITLE
light visual/order improvement to the explanation of content

### ISSUE THREAD

**Issue by LuckyLolo123** [freeCodeCamp/freeCodeCamp#45183]
## Affected page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-equality-operator

## Your code

```

no code


```

## Expected behavior

The student has to go all the way down up to the explanation "In order, these expressions would evaluate to true, false, true, and true." to realize what the result of the operations is.
My suggestion is that the second section of the code  should look like this:

1   ==  1    // true
1   ==  2    // false
1   == '1'   // true
"3" ==  3   // true

instead of the screenshot attached.

## Screenshots

![image](https://user-images.githubusercontent.com/92828033/154780662-13681fa8-30e0-4a8c-b15e-c6cf3feb74e4.png)

## System

- Device: [e.g. iPhone6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.4]
- Browser: [e.g. chrome, safari]
- Version: [e.g. 22]

**Comment by moT01** [comment]
I would be fine with this change. Any interest in making a PR for it @LuckyLolo123?

**Comment by moT01** [comment]
We have some [documentation that explains how to contribute](https://contribute.freecodecamp.org/#/how-to-setup-freecodecamp-locally). It may look a little intimidating, but if you follow the instructions it's not bad.

The file where this needs to be fixed is [here](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-equality-operator.md)

Technically, you can edit the file on github and make a PR from there - which can work for some simple fixes, but you can't see and test your changes. I think we would accept those if you want to do that, but I would recommend setting the project up locally at some point if you want to contribute regularly.

To be clear, we usually like to get second opinion before making changes like this - since we don't have that, I can't guarantee a PR would be merged - but I think others would be fine with the change so if you want to make a PR, go ahead.

**Comment by ojeytonwilliams** [comment]
This change seems good. For context, we got rid of a lot of comments since we'd need to translate them. However, these are keywords and don't need translation.

**Comment by Jagrutiti** [comment]
@ieahleen shall I take this up?

**Comment by ShaunSHamilton** [comment]
@nhcarrigan Could you clarify if this is actually what we wanted, because this is exactly what we removed 12 months ago in: https://github.com/freeCodeCamp/freeCodeCamp/pull/41092/files

**Comment by nhcarrigan** [comment]
Yes, this is okay because these instances specifically are only keywords that don't get translated anyway.
